### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by Ananya-m0140**
Should anything more be added like logo,etc?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,155 @@
...
+</a>
+<a href="https://twitter.com/metakgp">
+  <img align="center" alt="metakgp's Twitter " width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/twitter.svg" />
+</a>
+<a href="https://www.instagram.com/metakgp_/">
+  <img align="center" alt="metakgp's Instagram" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/instagram.svg" />
+</a>
+</p>
+
+# Credits
```
Comment (proffapt) [inline README.md]: https://github.com/metakgp/README.md/blob/256ba6d22b41617eb1b4223d12815260b038903d/README.md?plain=1#L159

Change the section like this. Credit @spookbite.

File: README.md [hunk 2]
```diff
@@ -1,54 +1,155 @@
...
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/f08c7a0a-1fd3-4c11-ab13-92bc120fea7f">
+
+  </a>
+</div>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+# Maintainer(s)
```
Comment (proffapt) [inline README.md]: https://github.com/metakgp/README.md/blob/256ba6d22b41617eb1b4223d12815260b038903d/README.md?plain=1#L154

Don't forget these one liner description for this and creator section as well.

File: README.md [hunk 3]
```diff
@@ -1,54 +1,155 @@
...
+
+<!-- MARKDOWN LINKS & IMAGES -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/metakgp/kronos2.0.svg?style=for-the-badge
+[contributors-url]: https://github.com/metakgp/kronos2.0/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/metakgp/kronos2.0.svg?style=for-the-badge
+[forks-url]: https://github.com/metakgp/kronos2.0/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/gyft.svg?style=for-the-badge
+[stars-url]: https://github.com/metakgp/kronos2.0/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/gyft.svg?style=for-the-badge
```
Comment (proffapt) [inline README.md]:
```suggestion
[issues-shield]: https://img.shields.io/github/issues/metakgp/kronos2.0.svg?style=for-the-badge
```

File: README.md [hunk 4]
```diff
@@ -1,54 +1,155 @@
...
+
+- [Arpit Bhardwaj](https://github.com/proffapt)
+
+<!-- MARKDOWN LINKS & IMAGES -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/metakgp/kronos2.0.svg?style=for-the-badge
+[contributors-url]: https://github.com/metakgp/kronos2.0/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/metakgp/kronos2.0.svg?style=for-the-badge
+[forks-url]: https://github.com/metakgp/kronos2.0/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/gyft.svg?style=for-the-badge
```
Comment (proffapt) [inline README.md]:
```suggestion
[stars-shield]: https://img.shields.io/github/stars/metakgp/kronos2.0.svg?style=for-the-badge
```

File: README.md [hunk 5]
```diff
@@ -1,54 +1,155 @@
...
+[stars-url]: https://github.com/metakgp/kronos2.0/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/gyft.svg?style=for-the-badge
+[issues-url]: https://github.com/metakgp/kronos2.0/issues
+[license-shield]: https://img.shields.io/github/license/metakgp/gyft.svg?style=for-the-badge
```
Comment (proffapt) [inline README.md]:
```suggestion
[license-shield]: https://img.shields.io/github/license/metakgp/kronos2.0.svg?style=for-the-badge
```

**Review by proffapt — CHANGES_REQUESTED** [review]
Keep the same order as in readme
- contact
- maintainer
- creator
- then markdown links

Don't deviate at all from the structure of readme. Also add other stuff, present in `.github` folder of the repository except workflows folder. Update LICENSE and add the section `additional documentation`

**Comment by Bikram-ghuku** [comment]
The docker setup method doesn't work since the updated code is not pushed to the dockerhub....

**Comment by proffapt** [comment]
> The docker setup method doesn't work since the updated code is not pushed to the dockerhub....

That will be done soon under #3

File: README.md [hunk 6]
```diff
@@ -1,54 +1,174 @@
...
+The currently active maintainer(s) of this project.
+- [Arpit Bhardwaj](https://github.com/proffapt)
+
+### Creator(s)
+
+Honoring the original creator(s) and ideator(s) of this project.
+
+<!-- UPDATE -->
+- [Palkit Lohia](https://github.com/spookbite)
+
```
Comment (proffapt) [inline README.md:147]:
```suggestion

<p align="right">(<a href="#top">back to top</a>)</p>

```

File: README.md [hunk 7]
```diff
@@ -1,54 +1,174 @@
...
+## About The Project
 
 This webapp displays previous year's grade distribution. It has been hosted with streamlit as an app but is still under development.
 
 You can go [here](https://kronos.streamlit.app/) for the live version of the project. Make your choices wisely :)
 
 You can find the container image [here](https://hub.docker.com/repository/docker/spookbite/kronosv2).
 
-# Setup 
+# Getting started
```
Comment (proffapt) [inline README.md]:
```suggestion
## Getting started
```

File: README.md [hunk 8]
```diff
@@ -1,54 +1,174 @@
...
 
-## (Without Docker)
+### (Without Docker)
```
Comment (proffapt) [inline README.md]:
```suggestion
### Without docker
```

File: README.md [hunk 9]
```diff
@@ -1,54 +1,174 @@
...
+  ```
 
-## Using Docker
+ 5. Run the following code on your terminal:
+  ```shell
+      streamlit run app.py
+  ```
 
-### Pull the Docker image
+### Using Docker
```
Comment (proffapt) [inline README.md]:
```suggestion
### Using docker
```

File: README.md [hunk 10]
```diff
@@ -1,54 +1,174 @@
...
+
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
 
 This webapp displays previous year's grade distribution. It has been hosted with streamlit as an app but is still under development.
 
 You can go [here](https://kronos.streamlit.app/) for the live version of the project. Make your choices wisely :)
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -1,54 +1,174 @@
...
+## Usage
+<!-- UPDATE -->
+Following is an example how this project works with streamlit.
+
+<div align="center">
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/37b7a1cf-0956-4545-9f31-6c45cc08b978">
+    </a>
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/f08c7a0a-1fd3-4c11-ab13-92bc120fea7f">
```
Comment (proffapt) [inline README.md]: Update these `PROJECT_NAME` values

**Review by proffapt — CHANGES_REQUESTED** [review]
Files are in wrong folder. They should be in `.github` not in `.github/workflows`
Also, i don't think that you have added all the files. 
image

- Add metaKGP logo as in that readme template
- rename it to kronos
- update all links from `kronos2.0` to `kronos`

**Review by proffapt — APPROVED** [review]

**Pull request merged**